modules/enet/doc_classes/NetworkedMultiplayerENet.xml

```diff
@@ -116,6 +116,9 @@
 			The compression method used for network packets. These have different tradeoffs of compression speed versus bandwidth, you may need to test which one works best for your use case if you use compression at all.
 		</member>
 		<member name="refuse_new_connections" type="bool" setter="set_refuse_new_connections" getter="is_refusing_new_connections" override="true" default="false" />
+		<member name="server_relay" type="bool" setter="set_server_relay_enabled" getter="is_server_relay_enabled" default="true">
+			Enable or disable the server feature that notify clients of other peers connection/disconnection, and relay messages between them. When this option is [code]false[/code] clients will not be automatically notified of other peers, nor be able to send them packets through the server.
```

Comment:
Suggestion for better grammar:

```suggestion
			Enable or disable the server feature that notify clients of other peers connection/disconnection, and relay messages between them. When this option is [code]false[/code], clients will not be automatically notified of other peers and won't be able to send them packets through the server.
```

(I think "nor" only makes sense in "neither … nor …", but I'm not 100% sure. Either way, this makes it more obvious that both actions are disabled.)

Reply:
Thanks, fixed